com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -163,6 +163,16 @@ public RenderBufferStoreAction depthStoreAction
             get => m_DepthStoreAction;
         }
 
+        public bool[] overriddenColorStoreActions
+        {
+            get => m_OverriddenColorStoreActions;
+        }
+
+        public bool overriddenDepthStoreAction
```

Comment:
I'm failing to understand why this public API is needed.

Reply:
the fact that it is public is an error, will change that to internal. 
Regarding why the overridden flag is used I moved the conversation to slack

Reply:
changed to internal in CS 5254f56